ios/titanium.xcconfig

```diff
@@ -4,11 +4,11 @@
 // OF YOUR TITANIUM SDK YOU'RE BUILDING FOR
 //
 //
-TITANIUM_SDK_VERSION = 9.0.0.GA
+TITANIUM_SDK_VERSION = 9.2.0
 
 //
 // THESE SHOULD BE OK GENERALLY AS-IS
 //
 TITANIUM_SDK = /Users/$(USER)/Library/Application Support/Titanium/mobilesdk/osx/$(TITANIUM_SDK_VERSION)
 HEADER_SEARCH_PATHS = $(inherited) "$(TITANIUM_SDK)/iphone/include"
-FRAMEWORK_SEARCH_PATHS = $(inherited) "$(TITANIUM_SDK)/iphone/Frameworks"
+FRAMEWORK_SEARCH_PATHS = $(inherited) "$(TITANIUM_SDK)/iphone/Frameworks/**"
```

Comment:
I remember that I faced some issue with this while building module (probably it was picking other architecture found inside TitaniumKit.xcframework e.g building for simulator it was picking device's framework) because it is searching framework recursively. But it is working fine now. May be some issue from my side.